--- liboggz-1.0.0/src/tools/Makefile.am.orig	2009-09-10 16:00:19.000000000 +0200
+++ liboggz-1.0.0/src/tools/Makefile.am	2009-10-01 07:07:43.312606747 +0200
@@ -75,9 +75,9 @@
 # see http://lists.xiph.org/pipermail/ogg-dev/2008-July/001083.html
 install-exec-local:
 	for p in dump diff info merge rip ; do \
-	  if test -e $(exec_prefix)/bin/oggz$$p ; then \
-	    $(RM) $(exec_prefix)/bin/oggz$$p ; \
-	    $(LN_S) $(exec_prefix)/bin/oggz-$$p $(exec_prefix)/bin/oggz$$p ; \
+	  if test -e $(DESTDIR)$(exec_prefix)/bin/oggz$$p ; then \
+	    $(RM) $(DESTDIR)$(exec_prefix)/bin/oggz$$p ; \
+	    $(LN_S) oggz-$$p $(DESTDIR)$(exec_prefix)/bin/oggz$$p ; \
 	  fi ; \
 	done
 
@@ -89,7 +89,7 @@
 # such systems.
 uninstall-local:
 	for p in dump diff info merge rip ; do \
-	  if test -L $(exec_prefix)/bin/oggz$$p ; then \
-	    $(RM) $(exec_prefix)/bin/oggz$$p ; \
+	  if test -L $(DESTDIR)$(exec_prefix)/bin/oggz$$p ; then \
+	    $(RM) $(DESTDIR)$(exec_prefix)/bin/oggz$$p ; \
 	  fi ; \
 	done
